pkg/client/access_control_records_object_set.go

```diff
@@ -3,68 +3,69 @@
 package client
 
 import (
-	"reflect"
 	"fmt"
 	"github.com/hpe-storage/common-host-libs/jsonutil"
-	"github.com/hpe-storage/nimble-golang-sdk/pkg/client/v1/model"
-	"github.com/hpe-storage/nimble-golang-sdk/pkg/util"
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/client/v1/nimbleos"
```

Comment:
That is true but at some point we have to make this change and it will be always like this. If you need more context I can help you with that it will help you to follow the changeset. It looks huge because of multiple nimble objects. You just follow one object change i.e volume, flow would be volume_service.go (it uses volume.go)-> volumes_object_set.go -> group_mgmt_client.go . Similarly for all the objects.

Reply:
I agree that renaming the packages will be a big change however it is done, but it will be a purely mechanical change.  The difficulty arises when mechanical changes are intermingled with other changes, making the other changes hard to find.  I thought that's what is happening here, but perhaps I misunderstand.